packages/hardhat/contracts/core/Controller.sol

```diff
@@ -476,7 +476,7 @@ contract Controller is Ownable, ReentrancyGuard, IERC721Receiver {
             return 0;
         }
 
-        // add back the bounty amount, liquidators only get reward from liquidation
+        // add back the bounty amount, liquidators onlly get reward from liquidation
```

Comment:
```suggestion
        // add back the bounty amount, liquidators only get reward from liquidation
```

Reply:
I think this maybe should be a separate PR and not merged in yet to ensure we have a commit that matches the deployed bytecode.  We can fix in master afterwards.

Reply:
i believe this is on my branch when i actually deployed.